src/domain/opendatafabric/src/dtos/dtos_extra.rs

```diff
@@ -90,6 +90,7 @@ impl Transform {
 ////////////////////////////////////////////////////////////////////////////////
 
 impl Default for SetVocab {
+    #[allow(clippy::derivable_impls)]
```

Comment:
Can we replace these with just one `!#[allow(clippy::derivable_impls)]` annotation on the module level?

Reply:
thought about it, but I'm worried that it will mask in new (potential places) in this file -- so I suggest leaving this "surgical" (or just verbose) approach here

what do you think about this?

Reply:
I think as long as the list of of ignored warnings doesn't grow - its OK to have it on module level. Preferring `derive(Default)` to `impl Default` is a very minor and harmless thing, so I'm not worried about it spreading. 

I actually prefer the explicit `impl` in many cases, especially in DTOs that carry configuration, as it forces developer who adds a property to think what the actual default should be.